### PR TITLE
docs: create metadata

### PR DESCRIPTION
**Pull request by jducnuigeen**
Fixes: #[243](https://github.com/linz/qgis-buildingsgeneralisation-plugin/issues/243)
Made to be as concise as possible, with the idea the Data Dictionary is more detailed.
Check attribution in the section "Metadata Constraints" since this has more than one attributing source.
Metadata File identifier created from ANZMetLite.

### CODE REVIEW

File: metadata/nz_facilities.xml [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+      </gmd:citation>
+      <gmd:abstract>
+        <gco:CharacterString>
+This dataset provides boundaries of facilities, currently hospitals and schools, within mainland New Zealand originally sourced in early 2021 from a combination of NationalMap and authoritative sources. Authoritative sources of these two types of attributes include the NZ Ministry of Health and NZ Ministry of Education.
+
+A facility polygon is a 2D representation of the boundary of a facility area as it appears to be currently used. Facility boundaries in this dataset were used to apply hospital and school building name and use to the NZ Building outlines dataset published on the LINZ Data Service.
+
+A more detailed description of NZ Facilities can be found in the NZ Facilities Data Dictionary [LINK]. This Data Dictionary also includes information on how NZ Facilities were used to support the attribution of NZ Building Outlines.
+
+NZ Facilities contains data sourced from NationalMap, Ministry of Education and Ministry of Health licensed for reuse under CC BY 4.0.
```
Comment (dwsilk) [inline metadata/nz_facilities.xml]:
```markdown
NZ Facilities contains data sourced from [NationalMap](https://www.nationalmap.co.nz/), [Ministry of Education](https://www.educationcounts.govt.nz/) and [Ministry of Health](https://www.health.govt.nz/) licensed for reuse under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).
```

Comment (dwsilk) [reply metadata/nz_facilities.xml]: Should contain links ^

File: metadata/nz_facilities.xml [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
+**Related NZ Facilities data**
```
Comment (dwsilk) [inline metadata/nz_facilities.xml]: Just `**Related data**` ?

File: metadata/nz_facilities.xml [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+            </gmd:MD_ScopeDescription>
+          </gmd:levelDescription>
+        </gmd:DQ_Scope>
+      </gmd:scope>
+      <gmd:lineage>
+        <gmd:LI_Lineage>
+          <gmd:statement>
+            <gco:CharacterString>
+**Source**
+NZ Facilities contains data sourced from NationalMap, Ministry of Education and Ministry of Health licensed for reuse under CC BY 4.0.
```
Comment (dwsilk) [inline metadata/nz_facilities.xml]: Same as above.. add links.

File: metadata/nz_facilities.xml [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+
+Coordinates for named hospitals and schools supplied by the Ministry of Health and the Ministry of Education were matched to facility boundaries from NationalMap and used to verify locations and names between these sources. LINZ aerial imagery was used to assist in manually correcting some boundary extents. The assigned boundaries were then used to attribute names and uses to groups of buildings in the NZ Buildings [LINK] dataset.
+
+**Attributes**
+• The FACILITY_ID provides a unique id for each facility.
+• The NAME attribute provides a facility name which has consistent naming convention applied. This name may directly match the name provide by the authoritative source.
```
Comment (dwsilk) [inline metadata/nz_facilities.xml]: provided (fix typo)

File: metadata/nz_facilities.xml [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+• The EXTERNAL_NAME attribute provides the facility name used by the authoritative source.
```
Comment (dwsilk) [inline metadata/nz_facilities.xml]: maybe this should be `source_name` ? probably too late to suggest that and I know that `external` matches the terminology we have used elsewhere..

**Review by dwsilk — CHANGES_REQUESTED** [review]

**Review by Douglas-K — APPROVED** [review]

**Pull request merged**